pkg/lang/ir/v1/compile.go

```diff
@@ -336,14 +336,11 @@ func (g *generalGraph) CompileLLB(uid, gid int) (llb.State, error) {
 		if err != nil {
 			return llb.State{}, errors.Wrap(err, "failed to compile entrypoint")
 		}
-		vscode, err := g.compileVSCode()
+		vscode, err := g.compileVSCode(entrypoint)
```

Comment:
I remember you mention that this should happen before the `run` command.

Reply:
Later I found that the issue of rust-analyzer not supporting the current platform is not due to `compileVSCode` first and then compile `run`. The rust-analyzer server checks rust every time it starts up, rather than checking it only once during installation. So it's not a matter.

Reply:
So will it be resolved to the correct platform?

Reply:
It's a seperate problem, I'll open a new issue and fix it.